recipes/server.rb

```diff
@@ -20,8 +20,14 @@
 
 include_recipe 'zabbix_ng::repository'
 
+#package 'zabbix-server-mysql'
```

Comment:
Just remove this. I don't see an advantage of using mysql, if we one day offer support for it, we should rather use an attribute that specifies the server package to install.